source/core/parameters.cc

```diff
@@ -382,6 +382,60 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  SurfaceTensionForce::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("surface tension force");
+    {
+      prm.declare_entry("output auxiliary fields",
+                        "false",
+                        Patterns::Bool(),
+                        "Output the phase fraction gradient and curvature");
+
+      prm.declare_entry(
+        "phase fraction gradient filter",
+        "0.5",
+        Patterns::Double(),
+        "The filter value for phase fraction gradient calculations to damp high-frequency errors");
+
+      prm.declare_entry(
+        "curvature filter",
+        "0.5",
+        Patterns::Double(),
+        "The filter value for curvature calculations to damp high-frequency errors");
+
+      prm.declare_entry(
+        "verbosity",
+        "quiet",
+        Patterns::Selection("quiet|verbose"),
+        "State whether from the surface tension force calculations should be printed "
+        "Choices are <quiet|verbose>.");
```

Comment:
Just out of curiosity, what is printed in this case?

Reply:
The details of the pfg and curvature solvers.